src/cljs/vanilla/widgets/make_chart.cljs

```diff
@@ -2,17 +2,22 @@
   (:require [reagent.core :as reagent]
             [re-frame.core :as rf]
             [day8.re-frame.tracing :refer-macros [fn-traced]]
-            [vanilla.widgets.util :as util]))
+            [vanilla.widgets.util :as util]
+            ["react-highcharts" :as ReactHighcharts]
+            ["highcharts-more" :as HighchartsMore]
+            [vanilla.dark-mode :as dark]))
 
 
+(HighchartsMore ReactHighcharts/Highcharts)
```

Comment:
This seems out of place

Reply:
we could consider moving it to sit next to the "module" attachments in core.cljs (start-dashboard). I put it here because this is the entry point for making ALL highchart types, and we need the "more" module hooked in. Or we could try moving he other module hooks to here?

Reply:
Hmm, I'm not sure if there's a good place for all of them that will work given that Sankey is "special". But I'd definitely make a note of the two outliers so that way when we look back at them in a month we don't say "Idk what it does but don't change it or everything breaks" - AKA how C++ dev goes.

Reply:
done. see https://github.com/cawasser/vanilla/pull/49/commits/2cc489cb61f41bdd7c44ea2633646579d56a61e3